src/command_modules/azure-cli-iotcentral/azure/cli/command_modules/iotcentral/_params.py

```diff
@@ -30,6 +30,10 @@ def load_arguments(self, _):
         c.argument('location', get_location_type(self.cli_ctx),
                    help='Location of your IoT Central application. Default is the location of target resource group.')
         c.argument('sku', arg_type=get_enum_type(AppSku),
-                   help='Pricing tier for IoT Central applications. Default value is F1, which is free.')
+                   help='Pricing tier for IoT Central applications. Default value is S1.')
```

Comment:
This is a breaking change. Can you add this change to the help to make sure it's more apparent?

Reply:
👍  on updating help

We've changed our billing model, S1 apps are now free up to 5 devices and F1 is no longer used. So there is a little more going on here than just a SKU change, and it would be useful to call this out in help or link to the pricing page the explains this.

Reply:
Done

Reply:
@williexu